src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -108,6 +108,7 @@ protected function createConfigurationDefinition()
             (new FixerOptionBuilder('case', 'Whether to camel or snake case when adding the test prefix'))
                 ->setAllowedValues(['camel', 'snake'])
                 ->setDefault('camel')
+                ->setDeprecationMessage('Use `php_unit_method_casing` fixer instead.')
```

Comment:
can `php_unit_method_casing` do same thing as this option ?

Reply:
Well, theoretically no because someone could want to transform `/** @test */ function fooBar` in `function test_fooBar` and only with this option you can do this.
But does this example make sense?

Reply:
then, if deprecated feature is not fully represented by new feature, the difference shall be mentioned as addition to "use X instead"

Reply:
Sounds right, but I have no idea what could I wrote more to explain the intentions.
If someone used to use *case* option for Annotation fixer, the only possibility that comes into my mind is that the developer is taking care of method casing in PHPUnit classes.
The name of the new fixer is self explanatory, as this is the exact purpose.

Reply:
well, actually, when I saw the new rule name, i though it's about proper casing of `assertsame` or `ExPeCtEdExCePtIoN`...